README.md

```diff
@@ -49,6 +49,16 @@ npm run preview-release
 # http://htmlpreview.github.io/?https://github.com/underdogio/underdogio.github.io/blob/{{preview_branch}}/index.html
 ```
 
+#### OSX Troubleshooting
+If you are on OSX and running into issues (e.g. `globstar` isn't defined and `sed` doesn't support `--in-place`), then please upgrade your `bash` and install `gnu-sed`:
+
+- `bash` upgrade available via shellshock test sites
```

Comment:
Ah, I forgot about that. I am surprised brew's `bash` comes first on the PATH but as long as it works =)